kedro/framework/cli/project.py

```diff
@@ -71,7 +71,15 @@ def project_group():  # pragma: no cover
 @forward_command(project_group, forward_help=True)
 @click.pass_obj  # this will pass the metadata as first argument
 def test(metadata: ProjectMetadata, args, **kwargs):  # pylint: disable=unused-argument
-    """Run the test suite."""
+    """Run the test suite. (DEPRECATED)"""
+
+    deprecation_message = (
+        "DeprecationWarning: Command 'kedro test' is deprecated and "
+        "will not be available from Kedro 0.19.0. "
+        "Use the command 'pytest' instead. "
```

Comment:
Just to be clear, this means we are not wrapping `pytest` in a Kedro command, but we're still supporting `pytest`, right? Since, if we were just going to say "testing is up to you now," we shouldn't call out `pytest` in the message.

I think it's fine as-is, if I've understood correctly.

Reply:
I think we will still have some tests in starters? This isn't too important though, as this deprecating message is for 0.18.x, we will still support `pytest` until 0.19.0, so this message is just asking people to use `pytest` instead of `kedro test`.
#1879 is a more detailed docs for how to use `pytest`.

Reply:
Determining whether or not tests will still be in starters is part of an open issue (#1849).

Reply:
I personally wouldn't mention `pytest` as an alternative in this message, I think this also would keep things consistent with the linting message.

Reply:
I am gonna leave it as is. I don't think it matters too much and I am fine with either option.

I checked in 0.17.x we did similar thing for `kedro install`
https://github.com/kedro-org/kedro/blob/80c0d3a47166c54f2bff4bed42cf63e06e65273c/kedro/framework/cli/project.py#L175-L180